Formula/n2n.rb

```diff
@@ -0,0 +1,25 @@
+class N2n < Formula
+  desc "Peer-to-peer VPN"
+  homepage "https://www.ntop.org/products/n2n/"
+  url "https://github.com/ntop/n2n/archive/2.4.tar.gz"
+  sha256 "acbf5792935b84fb6516b9a2133a0f6f70023ee6ee4ca0d2d4248cab187f3c04"
+
+  depends_on "openssl"
+
+  def install
+    system "make"
+    system "make", "test"
```

Comment:
Why are the test needed?

Reply:
Not really needed for it to function. It's more of a check to see if that compiles, but I do agree it is of not much use.
I added that just because I found other existing formulea are also compiling their tests.